### ISSUE TITLE
transitionDuration already defined with incompatible format

### ISSUE THREAD

**Issue by SudoPlz** [Instabug/instabug-reactnative#173]
This is a compile time issue. It just won’t let me build at all.

Here's the full stacktrace:

```
AGPBI: {"kind":"error","text":"Attribute \"transitionDuration\" already defined with incompatible format.","sources":[{"file":"/Users/isudoplz/.gradle/caches/transforms-1/files-1.1/instabug-core-4.15.2.aar/2e1bafa76cd6bc3636b83ee31e5468f1/res/values/values.xml","position":{"startLine":91,"startColumn":4,"startOffset":6720,"endColumn":51,"endOffset":6767}}],"original":"","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Original attribute defined here.","sources":[{"file":"/Users/isudoplz/.gradle/caches/transforms-1/files-1.1/instabug-core-4.15.2.aar/2e1bafa76cd6bc3636b83ee31e5468f1/res/values/values.xml","position":{"startLine":91,"startColumn":4,"startOffset":6720,"endColumn":51,"endOffset":6767}}],"original":"","tool":"AAPT"}
/Users/isudoplz/Development/Acuity/mobile/android/app/build/intermediates/res/merged/debug/values/values.xml:1943: error: Attribute "transitionDuration" already defined with incompatible format.
/Users/isudoplz/Development/Acuity/mobile/android/app/build/intermediates/res/merged/debug/values/values.xml:1768: Original attribute defined here.

Failed to execute aapt
com.android.ide.common.process.ProcessException: Failed to execute aapt
	at com.android.builder.core.AndroidBuilder.processResources(AndroidBuilder.java:796)
	at com.android.build.gradle.tasks.ProcessAndroidResources.invokeAaptForSplit(ProcessAndroidResources.java:551)
	at com.android.build.gradle.tasks.ProcessAndroidResources.doFullTaskAction(ProcessAndroidResources.java:285)
	at com.android.build.gradle.internal.tasks.IncrementalTask.taskAction(IncrementalTask.java:109)
	at sun.reflect.GeneratedMethodAccessor611.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:73)
	at org.gradle.api.internal.project.taskfactory.DefaultTaskClassInfoStore$IncrementalTaskAction.doExecute(DefaultTaskClassInfoStore.java:173)
	at org.gradle.api.internal.project.taskfactory.DefaultTaskClassInfoStore$StandardTaskAction.execute(DefaultTaskClassInfoStore.java:134)
	at org.gradle.api.internal.project.taskfactory.DefaultTaskClassInfoStore$StandardTaskAction.execute(DefaultTaskClassInfoStore.java:121)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.run(ExecuteActionsTaskExecuter.java:122)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:197)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:107)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:111)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:92)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:70)
	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:63)
	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:58)
	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:88)
	at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:52)
	at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:52)
	at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)
	at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
	at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:34)
	at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker$1.run(DefaultTaskGraphExecuter.java:248)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:197)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:107)
	at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:241)
	at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:230)
	at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.processTask(DefaultTaskPlanExecutor.java:124)
	at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.access$200(DefaultTaskPlanExecutor.java:80)
	at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:105)
	at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:99)
	at org.gradle.execution.taskgraph.DefaultTaskExecutionPlan.execute(DefaultTaskExecutionPlan.java:625)
	at org.gradle.execution.taskgraph.DefaultTaskExecutionPlan.executeWithTask(DefaultTaskExecutionPlan.java:580)
	at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.run(DefaultTaskPlanExecutor.java:99)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.util.concurrent.ExecutionException: com.android.ide.common.process.ProcessException: Error while executing process /Users/isudoplz/Library/Android/sdk/build-tools/27.0.3/aapt with arguments {package -f --no-crunch -I /Users/isudoplz/Library/Android/sdk/platforms/android-27/android.jar -M /Users/isudoplz/Development/Acuity/mobile/android/app/build/intermediates/manifests/full/debug/AndroidManifest.xml -S /Users/isudoplz/Development/Acuity/mobile/android/app/build/intermediates/res/merged/debug -m -J /Users/isudoplz/Development/Acuity/mobile/android/app/build/generated/source/r/debug -F /Users/isudoplz/Development/Acuity/mobile/android/app/build/intermediates/res/debug/resources-debug.ap_ -D /Users/isudoplz/Development/Acuity/mobile/android/app/build/intermediates/multi-dex/debug/manifest_keep.txt --custom-package com.sudoplz.app.android -0 apk --output-text-symbols /Users/isudoplz/Development/Acuity/mobile/android/app/build/intermediates/symbols/debug --no-version-vectors}
	at com.google.common.util.concurrent.AbstractFuture.getDoneValue(AbstractFuture.java:503)
	at com.google.common.util.concurrent.AbstractFuture.get(AbstractFuture.java:482)
	at com.google.common.util.concurrent.AbstractFuture$TrustedFuture.get(AbstractFuture.java:79)
	at com.android.builder.core.AndroidBuilder.processResources(AndroidBuilder.java:794)
	... 47 more
Caused by: com.android.ide.common.process.ProcessException: Error while executing process /Users/isudoplz/Library/Android/sdk/build-tools/27.0.3/aapt with arguments {package -f --no-crunch -I /Users/isudoplz/Library/Android/sdk/platforms/android-27/android.jar -M /Users/isudoplz/Development/Acuity/mobile/android/app/build/intermediates/manifests/full/debug/AndroidManifest.xml -S /Users/isudoplz/Development/Acuity/mobile/android/app/build/intermediates/res/merged/debug -m -J /Users/isudoplz/Development/Acuity/mobile/android/app/build/generated/source/r/debug -F /Users/isudoplz/Development/Acuity/mobile/android/app/build/intermediates/res/debug/resources-debug.ap_ -D /Users/isudoplz/Development/Acuity/mobile/android/app/build/intermediates/multi-dex/debug/manifest_keep.txt --custom-package com.sudoplz.app.android -0 apk --output-text-symbols /Users/isudoplz/Development/Acuity/mobile/android/app/build/intermediates/symbols/debug --no-version-vectors}
	at com.android.build.gradle.internal.process.GradleProcessResult.buildProcessException(GradleProcessResult.java:73)
	at com.android.build.gradle.internal.process.GradleProcessResult.assertNormalExitValue(GradleProcessResult.java:48)
	at com.android.builder.internal.aapt.AbstractProcessExecutionAapt$1.onSuccess(AbstractProcessExecutionAapt.java:78)
	at com.android.builder.internal.aapt.AbstractProcessExecutionAapt$1.onSuccess(AbstractProcessExecutionAapt.java:74)
	at com.google.common.util.concurrent.Futures$CallbackListener.run(Futures.java:1237)
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
	at com.google.common.util.concurrent.AbstractFuture.executeListener(AbstractFuture.java:911)
	at com.google.common.util.concurrent.AbstractFuture.complete(AbstractFuture.java:822)
	at com.google.common.util.concurrent.AbstractFuture.set(AbstractFuture.java:664)
	at com.google.common.util.concurrent.SettableFuture.set(SettableFuture.java:48)
	at com.android.build.gradle.internal.process.GradleProcessExecutor$1.run(GradleProcessExecutor.java:58)
Caused by: org.gradle.process.internal.ExecException: Process 'command '/Users/isudoplz/Library/Android/sdk/build-tools/27.0.3/aapt'' finished with non-zero exit value 1
	at org.gradle.process.internal.DefaultExecHandle$ExecResultImpl.assertNormalExitValue(DefaultExecHandle.java:380)
	at com.android.build.gradle.internal.process.GradleProcessResult.assertNormalExitValue(GradleProcessResult.java:46)
	... 9 more

 FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugResources'.
> Failed to execute aapt

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 2m 58s
```

Looks like the name `transitionDuration` is used two times:

Once here:
<img src="https://cl.ly/0j1d0c141Y1w/Image%202018-07-02%20at%2010.40.05%20PM.png" />

and another time here:
<img src="https://cl.ly/0j1d0c141Y1w/Image%202018-07-02%20at%2010.40.05%20PM.png" />

Tested and it happens both on instabug-reactnative 2.13.2 and on 2.13.3
Android Studio
react-native 0.55.4
development (build time)
aapt2 disabled
gradle 3.0.0

**Comment by SudoPlz** [comment]
@Korazy Hi Youssef, any news with that? Did you get a chance to take a look at that? It's currently blocking us :/

**Comment by Korazy** [comment]
Hey @SudoPlz 

Apologies for the delay, we're on it right now!

**Comment by SudoPlz** [comment]
@Korazy Sure, let me know if you find anything weird. On the mean time, keep in mind that on `2.11.2` that issue does not exist.

**Comment by Korazy** [comment]
@SudoPlz thanks for pointing that out :)

**Comment by Korazy** [comment]
Hey @SudoPlz 

I can see that Ziad is already helping you out with this over the support channel so I'll go ahead and close this issue :)_

**Comment by SudoPlz** [comment]
Yeah but other people might run into this.. Why not keep it open and post updates here?

**Comment by Korazy** [comment]
That also makes sense, I will make sure to communicate with Ziad and post the steps here when possible.

**Comment by HeshamMegid** [comment]
This issue is fixed starting from v8.0.1.

**Comment by SudoPlz** [comment]
I confirm that's fixed, thank you so much for that!
